runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/RunnerPCollectionView.java

```diff
@@ -35,6 +37,8 @@
 
 /** A {@link PCollectionView} created from the components of a {@link SideInput}. */
 class RunnerPCollectionView<T> extends PValueBase implements PCollectionView<T> {
+  public static final long serialVersionUID = 2465466441736952633L;
```

Comment:
make serialVersionUID private

The JVM reads this property directly and is not meant to be public.